Sources/TuistKit/Services/Inspect/InspecImplicitImportsService.swift

```diff
@@ -60,23 +60,23 @@ final class InspectImplicitImportsService {
 
     private func lint(graphTraverser: GraphTraverser) async throws -> [InspectImplicitImportsServiceErrorIssue] {
         let allTargets = graphTraverser
+            .allTargets()
```

Comment:
nit: You can get `allTargets` by making a union of `allInternalTargets()` and `allExternalTargets()`, so we don't loop through some targets unnecessarily twice